### ISSUE TITLE
Error code 357

### ISSUE THREAD

**Issue by SLornieCYC** [data-to-insight/csc-validator-be-903#498]
Current implementation returns too many false-positives:

```
eps = eps.loc[eps['DECOM'].notnull()]

first_eps = eps.loc[eps.groupby('CHILD')['DECOM'].idxmin()]
errs = first_eps[first_eps['RNE'] != 'S'].index.to_list()
```

This means any child with an open episode at `<COLLECTION_START_DATE>` will flag as an error unless the open episode (from the previous year) happens to have `<RNE>` = S.

Suggest this rule could be modified for the tool to :

- Only test first episodes where the `<DECOM>` is also on or after `<COLLECTION_START_DATE>`
- ~~Test for episodes where `<RNE>` != S **and** without an immediately preceding episode (ep-1 `<DEC>` = `<DECOM>`) [Although this would be a greater deviation from the original rule logic].~~ [EDIT: Think this bit is already covered by rule 433]

**Comment by tab1tha** [comment]
Has this been solved already?

**Comment by dezog** [comment]
fixed by 44d295d8f3761f6b8c4d581f1eeb4d7dfa9cdeb1
